examples/Basics/callable.py

```diff
@@ -0,0 +1,92 @@
+## E2E Test for using callables
```

Comment:
we can change it at a later time but i think Callables / Method-Hooks would certainly deserve their own section instead of `Basics` right?

Reply:
Are we planing to have more examples? I thought that the probabillity of improvement example will then replace this  one.

Reply:
yes I think at least feature importance is another example and likely others will emerge,
the file and its name is fine and can change later as you suggest, but my comment is more about the location being in `Basics`

Reply:
Okay, so I will make a `callable` folder for all future callable use cases and examples 👍

Reply:
since the thing is now called `hooks` which I agree is a little more precise perhaps the folder should also be named `Custom Hooks` ?

Reply:
yes sure, `Custom Hooks` sounds more specific 👍